### ISSUE TITLE
Failure: StressSpec, must down partitioned nodes, timeout

### ISSUE THREAD

**Issue by seglo** [akka/akka#29835]
https://jenkins.akka.io:8498/job/akka-nightly-jdk8/8233/consoleFull

```
[info] [JVM-12] - must down partitioned nodes (on node 'node-12', class akka.cluster.StressMultiJvmNode8) *** FAILED *** (3 seconds, 57 milliseconds)
[info] [JVM-12]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsgClass waiting for class akka.actor.ActorIdentity
[info] [JVM-12]   at scala.Predef$.assert(Predef.scala:223)
[info] [JVM-12]   at akka.testkit.TestKitBase.expectMsgClass_internal(TestKit.scala:569)
[info] [JVM-12]   at akka.testkit.TestKitBase.expectMsgType(TestKit.scala:541)
[info] [JVM-12]   at akka.testkit.TestKitBase.expectMsgType$(TestKit.scala:540)
[info] [JVM-12]   at akka.testkit.TestKit.expectMsgType(TestKit.scala:969)
[info] [JVM-12]   at akka.cluster.StressSpec.clusterResultAggregator(StressSpec.scala:561)
[info] [JVM-12]   at akka.cluster.StressSpec.$anonfun$createResultAggregator$2(StressSpec.scala:552)
[info] [JVM-12]   at akka.remote.testkit.MultiNodeSpec.runOn(MultiNodeSpec.scala:375)
[info] [JVM-12]   at akka.cluster.StressSpec.createResultAggregator(StressSpec.scala:551)
[info] [JVM-12]   at akka.cluster.StressSpec.$anonfun$partitionSeveral$1(StressSpec.scala:716)
[info] [JVM-12]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[info] [JVM-12]   at akka.testkit.TestKitBase.within(TestKit.scala:416)
[info] [JVM-12]   at akka.testkit.TestKitBase.within$(TestKit.scala:404)
[info] [JVM-12]   at akka.testkit.TestKit.within(TestKit.scala:969)
[info] [JVM-12]   at akka.testkit.TestKitBase.within(TestKit.scala:431)
[info] [JVM-12]   at akka.testkit.TestKitBase.within$(TestKit.scala:431)
[info] [JVM-12]   at akka.testkit.TestKit.within(TestKit.scala:969)
[info] [JVM-12]   at akka.cluster.StressSpec.partitionSeveral(StressSpec.scala:712)
[info] [JVM-12]   at akka.cluster.StressSpec.$anonfun$new$19(StressSpec.scala:910)
[info] [JVM-12]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[info] [JVM-12]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[info] [JVM-12]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[info] [JVM-12]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info] [JVM-12]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info] [JVM-12]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike$$anon$3.apply(AnyWordSpecLike.scala:1076)
[info] [JVM-12]   at org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
[info] [JVM-12]   at org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
[info] [JVM-12]   at akka.cluster.StressSpec.akka$cluster$MultiNodeClusterSpec$$super$withFixture(StressSpec.scala:439)
[info] [JVM-12]   at akka.cluster.MultiNodeClusterSpec.withFixture(MultiNodeClusterSpec.scala:185)
[info] [JVM-12]   at akka.cluster.MultiNodeClusterSpec.withFixture$(MultiNodeClusterSpec.scala:181)
[info] [JVM-12]   at akka.cluster.StressSpec.withFixture(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.invokeWithFixture$1(AnyWordSpecLike.scala:1074)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$runTest$1(AnyWordSpecLike.scala:1086)
[info] [JVM-12]   at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.runTest(AnyWordSpecLike.scala:1086)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.runTest$(AnyWordSpecLike.scala:1068)
[info] [JVM-12]   at akka.cluster.StressSpec.org$scalatest$BeforeAndAfterEach$$super$runTest(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.BeforeAndAfterEach.runTest(BeforeAndAfterEach.scala:234)
[info] [JVM-12]   at org.scalatest.BeforeAndAfterEach.runTest$(BeforeAndAfterEach.scala:227)
[info] [JVM-12]   at akka.cluster.StressSpec.runTest(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$runTests$1(AnyWordSpecLike.scala:1145)
[info] [JVM-12]   at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:413)
[info] [JVM-12]   at scala.collection.immutable.List.foreach(List.scala:392)
[info] [JVM-12]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info] [JVM-12]   at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:390)
[info] [JVM-12]   at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:427)
[info] [JVM-12]   at scala.collection.immutable.List.foreach(List.scala:392)
[info] [JVM-12]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info] [JVM-12]   at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:396)
[info] [JVM-12]   at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:475)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.runTests(AnyWordSpecLike.scala:1145)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.runTests$(AnyWordSpecLike.scala:1144)
[info] [JVM-12]   at akka.cluster.StressSpec.runTests(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.Suite.run(Suite.scala:1112)
[info] [JVM-12]   at org.scalatest.Suite.run$(Suite.scala:1094)
[info] [JVM-12]   at akka.cluster.StressSpec.org$scalatest$wordspec$AnyWordSpecLike$$super$run(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$run$1(AnyWordSpecLike.scala:1190)
[info] [JVM-12]   at org.scalatest.SuperEngine.runImpl(Engine.scala:535)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.run(AnyWordSpecLike.scala:1190)
[info] [JVM-12]   at org.scalatest.wordspec.AnyWordSpecLike.run$(AnyWordSpecLike.scala:1188)
[info] [JVM-12]   at akka.cluster.StressSpec.org$scalatest$BeforeAndAfterAll$$super$run(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:213)
[info] [JVM-12]   at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
[info] [JVM-12]   at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
[info] [JVM-12]   at akka.cluster.StressSpec.run(StressSpec.scala:439)
[info] [JVM-12]   at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)
[info] [JVM-12]   at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1320)
[info] [JVM-12]   at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1314)
[info] [JVM-12]   at scala.collection.immutable.List.foreach(List.scala:392)
[info] [JVM-12]   at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1314)
[info] [JVM-12]   at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:993)
[info] [JVM-12]   at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:971)
[info] [JVM-12]   at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1480)
[info] [JVM-12]   at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:971)
[info] [JVM-12]   at org.scalatest.tools.Runner$.main(Runner.scala:775)
[info] [JVM-12]   at org.scalatest.tools.Runner.main(Runner.scala)
```

**Comment by chbatey** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-2.13/737/console

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-jdk8/8566/consoleText

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-jdk8/8726/consoleText

Not quite the same though: "must leave nodes one-by-one from large cluster"

**Comment by patriknw** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-jdk8/8797

```
must leave nodes one-by-one from large cluster (on node 'node-6', class akka.cluster.StressMultiJvmNode2) *** FAILED *** (24 seconds, 48 milliseconds)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[[34mJVM-6[0m] [31m  java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsgClass waiting for class akka.actor.ActorIdentity
```

**Comment by raboof** [comment]
https://jenkins.akka.io:8498/job/akka-nightly-jdk8/8969/consoleFull

**Comment by patriknw** [comment]
fixed in #30628
